pkg/cloud/services/loadbalancer/loadbalancer.go

```diff
@@ -484,47 +471,6 @@ func checkIfLbExists(client *gophercloud.ServiceClient, name string) (*loadbalan
 	return &lbList[0], nil
 }
 
-func getOrCreateFloatingIP(client *gophercloud.ServiceClient, openStackCluster *infrav1.OpenStackCluster, ip string) (*floatingips.FloatingIP, error) {
-	var fp *floatingips.FloatingIP
-	var err error
-	if ip != "" {
-		fp, err = checkIfFloatingIPExists(client, ip)
```

Comment:
this logic is removed, where we cover this logic after the refactor?

Reply:
This logic was duplicated.
https://github.com/kubernetes-sigs/cluster-api-provider-openstack/blob/master/pkg/cloud/services/networking/floatingip.go#L32

Reply:
ok, got it